pkg/cilium/types.go

```diff
@@ -17,19 +17,26 @@ package cilium
 import "path/filepath"
 
 const (
+	// Name defines the extension name.
```

Comment:
// Name defines the networking extension name.

Reply:
I think we don't have to specify network since its already implicitly inferred from the repo.